server/test/queries/selectUsers.js

```diff
@@ -0,0 +1,27 @@
+const test = require('tape');
+
+const connection = require('../../database/config/connection');
+const {
+  buildDb,
+  buildFakeData,
+  buildStaticData,
+} = require('../../database/config/build');
+const selectUser = require('../../database/queries/selectUser');
+
+const selectId = () => connection.query('SELECT id from "user" limit 1');
```

Comment:
i think you don't need to this function, because you know what's inside the fake data, so you know there is a row have a id equal 1
```suggestion
const selectId = () => connection.query('SELECT id from "user" limit 1');
```